.github/workflows/file_read_write_round_trip_verification/test_write_read.py

```diff
@@ -0,0 +1,47 @@
+#  Copyright (c) 2023. Lance Developers
```

Comment:
Any chance we can treat this like the integtest so the test script is in the tests directory but only run if asked for?

Reply:
fixed, made it run on `workflow_dispatch`